internal/pkg/provision/access/access.go

```diff
@@ -0,0 +1,6 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Package access provides methods to access provisioned Talos cluster.
```

Comment:
Not something for this PR, but we need to do this a lot more. Also, a convention I have seen is to call this `docs.go`. Or was it `doc.go` 🤔? Something for later.

Reply:
I tried to follow the convention of having `<package_name>/<package_name>.go` file with this documentation line and most common types.